modules/home/assets/js/components/site-settings-list-item.js

```diff
@@ -0,0 +1,89 @@
+import ListItem from '@elementor/ui/ListItem';
+import ListItemText from '@elementor/ui/ListItemText';
+import Link from '@elementor/ui/Link';
+import Dialog from '@elementor/ui/Dialog';
+import Button from '@elementor/ui/Button';
+import DialogActions from '@elementor/ui/DialogActions';
+import DialogContent from '@elementor/ui/DialogContent';
+import DialogContentText from '@elementor/ui/DialogContentText';
+import DialogHeader from '@elementor/ui/DialogHeader';
+import DialogHeaderGroup from '@elementor/ui/DialogHeaderGroup';
+import DialogTitle from '@elementor/ui/DialogTitle';
+import Box from '@elementor/ui/Box';
+import TextField from '@elementor/ui/TextField';
+
+const SiteSettingsListItem = ( { item, image } ) => {
+	const [ open, setOpen ] = React.useState( false );
+	const [ pageName, setPageName ] = React.useState( '' );
+
+	const handlePopupOpen = ( e ) => {
+		if ( ! item.new_page ) {
+			return;
+		}
+
+		e.preventDefault();
+		setOpen( true );
+	};
+
+	const handlePopupClose = () => setOpen( false );
+
+	const handleChange = ( e ) => {
+		const encodedPostTitle = encodeURIComponent( e.target.value );
+		const postData = { post_title: encodedPostTitle };
+		const urlParams = new URLSearchParams( postData );
+
+		setPageName( urlParams.toString() );
+	};
+
+	return (
+		<ListItem alignItems="flex-start" sx={ { gap: 1, p: 0, maxWidth: '150px' } }>
```

Comment:
Could we refactor that, so that the markup and styling is shared with the other list items? I fear that in this way we have to update the markup of the list items in two separate place.